packages/tradershub/src/modals/RegulationModal/Row.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { qtMerge } from '@deriv/quill-design';
+import { Text, TextProps } from '@deriv-com/ui/dist/components/Text';
+import { TRegulatorsContentProps, TRowItem } from '../../constants/regulators-modal-content';
+
+const Row = ({ attribute, content, id, idx }: TRegulatorsContentProps) => (
+    <tr className={qtMerge('min-h-2000', idx === 0 && 'bg-brand-pink-light')} key={id}>
+        <td
+            className={`sticky z-10 align-middle border-solid start-50 py-500 px-400 border-system-light-active-background border-x-75 border-b-75 ${
+                idx === 0 ? 'bg-brand-pink-light' : 'bg-system-light-primary-background'
+            }`}
+        >
+            <Text align='left' as='p' color='general' size='xs' weight='bold'>
+                {attribute}
+            </Text>
+        </td>
+        {Object.keys(content).map(rowKey => (
+            <td
+                className='flex-col justify-center align-middle border-solid whitespace-nowrap py-500 px-400 border-b-75 border-r-75 border-system-light-active-background'
+                key={rowKey}
+            >
+                {Array.isArray(content[rowKey]) ? (
+                    (content[rowKey] as TRowItem[])?.map(item => (
+                        <Text
+                            align={(item?.options?.align as TextProps['align']) ?? 'center'}
```

Comment:
@mahdiyeh-deriv I can't do this too, because Text component from deriv-com/ui needs `align` props should be `TextAlign | undefined`, but `align`, `weight` and `color` from `getCFDContents` and `getOptionsContents` are just `string`. I can add more types for it in `options` of `TRowItem`:

```
export type TRowItem = {
    options?: Record<string, boolean | string>;
    text: string;
};
```
but I'm not sure we really need it... Let me know you decision, and I'll add separate types for `align`, `weight`, `color` and `should_show_asterick_at_end`.

Reply:
I'm stupid :) solved this issue